pom.xml

```diff
@@ -135,7 +135,7 @@
         <dependency>
             <groupId>io.appium</groupId>
             <artifactId>java-client</artifactId>
-            <version>8.0.0</version>
```

Comment:
Maybe keep this is a patch, i.e., `8.0.1`? 🤔

Reply:
Yes, exactly my point

Reply:
wait, you mean 2.0.1, the 8.0.0 change is the Appium version that has the fix